### PR TITLE
Sets Gradle home and fixes minor doc issues

### PR DESCRIPTION
**Pull request by chanseokoh**
# Changes

- Properly set Gradle home with `--gradle-user-home`.
- Fix minor doc issues (`tekton.dev/v1alpha1` --> `tekton.dev/v1beta1`)

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [X] Yaml file complies with [yamllint](https://github.com/adrienverge/yamllint) rules.

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: jib-gradle/jib-gradle.yaml [hunk 1]
```diff
@@ -42,6 +42,7 @@ spec:
         --stacktrace --console=plain \
         --init-script=/tekton/home/init-script.gradle \
         -Duser.home=/tekton/home \
+        --gradle-user-home=/tekton/home/.gradle \
```
Comment (loosebazooka) [inline jib-gradle/jib-gradle.yaml:45]: do we need both these values? line 45 and 46?

Comment (chanseokoh) [reply jib-gradle/jib-gradle.yaml:45]: Maybe not, but just wanted to be on the safe side. BTW, `--gradle-user-home` is documented in `man gradle`, so I think this is the right way to do it. Do you think we should remove line 64?

Comment (loosebazooka) [reply jib-gradle/jib-gradle.yaml:45]: I have no idea, I guess it couldn't hurt.

**Comment by chanseokoh** [comment]
Closing as the changes are part of the merged #407.

**Pull request closed without merging**